serializer/samples/devicetwin_simplesample/devicetwin_simplesample.c

```diff
@@ -36,6 +36,8 @@ DECLARE_MODEL(CarState,
     WITH_REPORTED_PROPERTY(ascii_char_ptr, vanityPlate)
 );
 
+// NOTE: In order for the callback to be invoked you will need to comment out the
```

Comment:
I would suggest this text:
For callbacks defined in the serializer model to be fired for desired properties, IoTHubClient_SetDeviceTwinCallback must not be invoked for the same device in the same application.
Please comment out the call to IoTHubClient_SetDeviceTwinCallback further down to enable the callbacks defined in the model below.